server/recruitment.py

```diff
@@ -38,6 +38,15 @@ def recruiter_claim_applicant(recruiter_user_id, applicant_user_id):
     applicant.put()
     return {'status': 'ok'}
 
+def submit_application(applicant_user_id):
+    applicant = User.get(applicant_user_id)
+    if applicant is None:
```

Comment:
Remove this check from here for now. All role checking should be done in the routing file (`main.py` for now), and I'll implement it later.

Reply:
done, But if someone calls this endpoint it will add junk to the db now